### PR TITLE
Refactor log

### PR DESCRIPTION
**Pull request by alexb-bloxroute**

### CODE REVIEW

File: api/service.go [hunk 1]
```diff
@@ -364,28 +365,35 @@ func (s *Service) GetPayload(ctx context.Context, receivedAt time.Time, payload
 }
 
 type ErrorResp struct {
-	Code        int    `json:"code"`
-	Message     string `json:"message"`
```
Comment (bhakiyakalimuthu) [inline api/service.go:368]: Keep this Message as it is. This is for the validator to respond. Include only relay & proxy message inside blxr message

Comment (alexb-bloxroute) [reply api/service.go:368]: Fixed

File: api/service.go [hunk 2]
```diff
@@ -97,15 +98,15 @@ func (s *Service) RegisterValidator(ctx context.Context, receivedAt time.Time, p
 			}
 			out, err := c.RegisterValidator(clientCtx, req)
 			if err != nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), id, "relay returned error", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), "", id, "relay returned error", clientIP)
 				return
 			}
 			if out == nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, "failed to register", id, "empty response from relay", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, "", "failed to register", id, "empty response from relay", clientIP)
```
Comment (bhakiyakalimuthu) [inline api/service.go:105]: do not need to respond this to follow same as relay
```suggestion
				errChan <- toErrorResp(http.StatusInternalServerError, "", "", id, "empty response from relay", clientIP)
```

File: api/service.go [hunk 3]
```diff
@@ -333,15 +334,15 @@ func (s *Service) GetPayload(ctx context.Context, receivedAt time.Time, payload
 
 			out, err := c.GetPayload(clientCtx, req)
 			if err != nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), id, "relay returned error", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), "", id, "relay returned error", clientIP)
 				return
 			}
 			if out == nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, "failed to getPayload", id, "empty response from relay", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, "", "failed to getPayload", id, "empty response from relay", clientIP)
```
Comment (bhakiyakalimuthu) [inline api/service.go:341]: same as above. just confirm relay once

File: api/service.go (hunk 2 above)
Comment (alexb-bloxroute) [reply api/service.go]: Fixed

File: api/service.go [hunk 4]
```diff
@@ -116,7 +117,7 @@ func (s *Service) RegisterValidator(ctx context.Context, receivedAt time.Time, p
 	for i := 0; i < len(s.clients); i++ {
 		select {
 		case <-ctx.Done():
-			return nil, nil, toErrorResp(http.StatusInternalServerError, "failed to register", id, ctx.Err().Error(), clientIP)
+			return nil, nil, toErrorResp(http.StatusInternalServerError, "", "failed to register", id, ctx.Err().Error(), clientIP)
```
[the remark this answers is not in the document]
Comment (alexb-bloxroute) [reply api/service.go]: Fixed. Yeah this should be empty.

File: api/service.go (hunk 3 above)
Comment (alexb-bloxroute) [reply api/service.go]: Fixed.

**Pull request merged**